src/components/ButtonForm/styles.module.scss

```diff
@@ -0,0 +1,27 @@
+@import 'variables/colors';
+
+.button {
+  border: 2px solid $green;
+  border-radius: 5px;
+  color: $green;
+  font-size: 16px;
+  padding: 7px 0;
+  text-align: center;
+  width: 100%;
+  margin: 20px auto;
+}
+
+.button:nth-of-type(2)::before {
+  content: '';
+  background: $gray-main;
+  height: 2px;
+  left: 0;
+  position: relative;
+  display: block;
+  top: -30px;
+}
```

Comment:
are you sure that you want this behaviour for all buttons of your app? what if in a future you need to put 3 buttons, the second will always have a separator line. What if you put this on signup component and send a classname to the reusable button?